src/main/java/ch/njol/skript/conditions/CondGlowingText.java

```diff
@@ -29,43 +32,79 @@
 import ch.njol.skript.doc.Since;
 import org.bukkit.block.BlockState;
 import org.bukkit.block.Sign;
+import org.bukkit.block.sign.Side;
 import org.bukkit.inventory.meta.BlockStateMeta;
 import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Locale;
 
 @Name("Has Glowing Text")
-@Description("Checks whether a sign (either a block or an item) has glowing text")
-@Examples("if target block has glowing text")
-@Since("2.8.0")
+@Description("Checks whether a sign (either a block or an item) has glowing text.")
+@Examples({
+	"if target block has glowing text:",
+	"if target block doesn't have glowing text on the back side:"
+})
+@Since("2.8.0, INSERT VERSION (front/back)")
 public class CondGlowingText extends PropertyCondition<Object> {
 
+	private static final boolean HAS_SIDES = Skript.classExists("org.bukkit.block.sign.Side");
+
 	static {
-		if (Skript.methodExists(Sign.class, "isGlowingText"))
-			register(CondGlowingText.class, PropertyType.HAVE, "glowing text", "blocks/itemtypes");
+		String sideChoice = " [on the (:front|:back) [side]]";
```

Comment:
```suggestion
		final String sideChoice = " [on the (:front|:back) [side]]";
```
The LDC can be inlined during compilation rather than using a variable

Reply:
It seems like this should go within the method exists check too.